client/src/services/features.tsx

```diff
@@ -0,0 +1,20 @@
+type Toggles = ReturnType<typeof getInitialFeatureToggles>;
+export type FeatureName = keyof Toggles;
+
+export let featureToggles: Toggles = getInitialFeatureToggles();
```

Comment:
I'm thinking about using something like that:

https://github.com/happykit/flags

what do you think?

Reply:
well.. it's sdk for SaaS service...

Reply:
meh, 5KB for just using boolean values..